docs/units-and-global-variables.rst

```diff
@@ -161,7 +161,6 @@ Members of string
 
 - ``string.concat(...) returns (string memory)``: :ref:`Concatenates variable number of string arguments to one string array<string-concat>`
 
-
```

Comment:
While I guess we could remove the newline here, it'd be nicer to keep the PR clean of unrelated changes like this :-).